security.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported versions
+
+Support for fallows lifecycle mentioned in [Canonical Kubernetes release cycle](https://ubuntu.com/about/release-cycle#canonical-kubernetes-release-cycle).
```

Comment:
This "Support for fallows" does not read right.

Reply:
```suggestion
Support follows the lifecycle mentioned in [Canonical Kubernetes release cycle](https://ubuntu.com/about/release-cycle#canonical-kubernetes-release-cycle).
```